tests/base/testcontainers.py

```
@@ -0,0 +1,32 @@
+        self.with_exposed_ports(TEMPORAL_GRPC_PORT, TEMPORAL_UI_PORT)
+        self.with_env("DB", "sqlite")
+        self.with_env("SQLITE_PRAGMA_journal_mode", "WAL")
+
+    def get_temporal_url(self) -> str:
+        host = self.get_container_host_ip()
+        port = self.get_exposed_port(TEMPORAL_GRPC_PORT)
+        return f"{host}:{port}"
+
+    def get_temporal_ui_url(self) -> str:
+        if not self._container:
+            raise RuntimeError("No temporal container")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)